release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1735,6 +1735,24 @@ If these values are not defined in a failure domain configuration, for instance
 [id="ocp-release-note-olm-bug-fixes_{context}"]
 ==== Operator Lifecycle Manager (OLM)
 
+* Previously, concurrent reconciliation of the same namespace in {olmv0-first} led to `ConstraintsNotSatisfiable` errors on subscriptions. This update resolves the issue. (link:https://issues.redhat.com/browse/OCPBUGS-48660[*OCPBUGS-48660*])
+
+* Previously, excessive catalog source snapshots caused severe performance regressions. This update fixes the issue. (link:https://issues.redhat.com/browse/OCPBUGS-48644[*OCPBUGS-48644*])
+
+* Previously, when kubelet terminated catalog registry pods with the "TerminationByKubelet" message, the registry pods were not recreated by the catalog Operator. This update fixes the issue. (link:https://issues.redhat.com/browse/OCPBUGS-46474[*OCPBUGS-46474*])
+
+* Previously, {olmv0} failed to upgrade Operator cluster service versions (CSVs) due to a TLS validation error. This update fixes the issue. (link:https://issues.redhat.com/browse/OCPBUGS-43581[*OCPBUGS-43581*])
+
+* Previously, service account tokens for Operator groups failed to generate automatically in {olmv0-first}. This update fixes the issue. (link:https://issues.redhat.com/browse/OCPBUGS-42360[*OCPBUGS-42360*])
+
+* Previously when {olmv1-first} validated custom resource definition (CRD) upgrades, the message output when detecting changed default values was rendered in bytes instead of human-readable language. With this update, related messages are now updated to show human-readable values. (link:https://issues.redhat.com/browse/OCPBUGS-41726[*OCPBUGS-41726*])
+
+* Previously, the status update function did not return an error when an error occurred in the Catalog Operator. As a result, the Operator might crash because the IP address returned a `nil` status. This update resolves the issue. (link:https://issues.redhat.com/browse/OCPBUGS-37637[*OCPBUGS-37637*])
```

Comment:
```suggestion
* Previously, the status update function did not return a connection error when an error occurred in the Catalog Operator. As a result, the Operator might crash because the IP address returned a `nil` status. This update resolves the issue. (link:https://issues.redhat.com/browse/OCPBUGS-37637[*OCPBUGS-37637*])
```
? problem and resolution don't seem =